db/schema.rb

```diff
@@ -75,6 +75,7 @@
     t.datetime "updated_at", null: false
     t.bigint "tenant_id"
     t.datetime "discarded_at"
+    t.string "workflow_ref"
```

Comment:
@syncrou is the workflow_ref always required? Should Auto Approval Workref be the default.

Reply:
@mkanoor There currently is no requirement to set a default workflow or default a workflow to auto approved, per my discussions with @bzwei.

Reply:
The workflow is not always required at this level as it could be inherited from the organization in the future.  I see this as a topic we can address in a separate PR.  This PR is purely focused on CRUD for the Portfolio.

Just to leave some thoughts on the topic of `PortfolioItem` that resolve to an empty workflow:  Maybe auto-approval or maybe the item is considered disabled and cannot be ordered?  We need to consider if there is a use-case for not specifying a workflow but allowing an item to be ordered.